docs/authoring.md

```diff
@@ -211,17 +211,17 @@ The `generator` property from `package.json` file must contain a JSON object tha
     }
   },
   "conditionalFiles": {
-    "src/api/adapters/amqp.js": {
+    "template/dir/relative/path/to/file/test-amqp.js": {
```

Comment:
Without `template` in the beginning, because of https://github.com/asyncapi/generator/blob/736bfa0108d158da07d86acaffbc57438c402f73/lib/generator.js#L546